frame/bounties/src/lib.rs

```diff
@@ -197,14 +226,29 @@ decl_storage! {
 
 		/// Bounties that have been made.
 		pub Bounties get(fn bounties):
-		map hasher(twox_64_concat) BountyIndex
-		=> Option<Bounty<T::AccountId, BalanceOf<T>, T::BlockNumber>>;
+			map hasher(twox_64_concat) BountyIndex
+			=> Option<Bounty<T::AccountId, BalanceOf<T>, T::BlockNumber>>;
 
 		/// The description of each bounty.
-		pub BountyDescriptions get(fn bounty_descriptions): map hasher(twox_64_concat) BountyIndex => Option<Vec<u8>>;
+		pub BountyDescriptions get(fn bounty_descriptions):
+			map hasher(twox_64_concat) BountyIndex => Option<Vec<u8>>;
 
 		/// Bounty indices that have been approved but not yet funded.
 		pub BountyApprovals get(fn bounty_approvals): Vec<BountyIndex>;
+
+		/// SubBounties that have been made.
+		pub SubBounties get(fn subbounties):
+			double_map hasher(twox_64_concat) BountyIndex,
+			hasher(twox_64_concat) BountyIndex =>
+			Option<SubBounty<T::AccountId, BalanceOf<T>, T::BlockNumber>>;
+
+		/// The SubBounties description of each subbounty.
+		pub SubBountyDescriptions get(fn subbounty_descriptions):
```

Comment:
No need to have this, just reuse `BountyDescriptions`

Reply:
I hope having separate description adds better details about subbounty for external world, instead of sharing parent bounty description for all sub-bounty. Plz kindly share your view.

Reply:
Every sub bounty should have their own unique bounty index, so you can store the description of sub bounty in `BountyDescriptions`

Reply:
Rework Done, subbounty description is part of BountyDescriptions & "SubBountyDescriptions" is removed